### ISSUE TITLE
Hyprland Config file errors

### ISSUE THREAD

**Issue by pranshunotpriyanshu** [yashlakhtariya/ysl-hyprdots#3]
![image](https://github.com/user-attachments/assets/b8a1e91b-06e7-4379-ad7b-895736664ca5)

how to fix these errors, and also, how to enable autohide for the nwg-dock, also I'm not able to get the dock styles but the normal nwg-dock
![image](https://github.com/user-attachments/assets/fb2f702c-51b4-4a37-bf76-66b1e6204038)
how to fix these errors @yashlakhtariya @yashslakhtariya

**Comment by yashlakhtariya** [comment]
You haven't installed hyprexpo plugin that's why it is showing error, either install it or remove its configuration from userprefs.conf

The nwg-dock is getting default style, to change theme, change value of $dock variable in userprefs.conf to preferred one, check some of my predefined configs in nwg-dock folder in .config and change margin, icon size, etc options in dock variable (prefer its --help) open some apps and right click on dock and pin them to dock, the pinned apps config is there in .cache directory names **nwg-dock-pinned** text file. 

For autohide dock, remove -x option and use -d or -r in dock variable (check their github readme for configuration options)

In short, you should prefer nwg-dock command options and configuration guide and for styling css, I have already kept my configs there in nwg-dock directory. If you know CSS, you can customize fully. And I recommend to install hyprexpo plugin, otherwise remove those config lines from userprefs.conf to remove this error.
